docs/get-started/totally-looks-like/index.md

```diff
@@ -0,0 +1,198 @@
+# Finetuning ResNet50 on Totally Looks Like Dataset
+
+In this tutorial, we will use Finetuner to finetune embeddings on [Totally Looks Like dataset](https://sites.google.com/view/totally-looks-like-dataset) (TLL) dataset.
+The dataset consists of 6016 pairs of images (12032 in total).
+
+>Totally-Looks-Like is a dataset and benchmark challenging machine-learned representations to reproduce human perception of image similarity.
+
+![overview](overview.png)
+
+*Each image patch in the left has a corresponded similar image patch in the right.*
+
+Finetuner adopts the idea of [transfer learning](https://en.wikipedia.org/wiki/Transfer_learning) and [metric learning](https://en.wikipedia.org/wiki/Similarity_learning#Metric_learning). The rationale is
+1. TLL dataset is a relatively small dataset, it's not reasonable to train a large network, such as ResNet on TLL from scratch. So we train part of the network by freezing layers.
+2. TLL dataset consists of pairs of images that can be formed as a positive pair (same classes), and a random image can be considered as a negative pair. We can form a `triplet` and use the Finetuner `TripletLoss`. We expect after fine-tuning, the distance between positive pairs can be pulled closer, while the distance between positive and negative pairs can be pushed away.
+
+## Environment & Data Preparation
+
+We will download `left.zip` and `right.zip`, as stated before,
+each of them consists of 6016 images which can be formed into pairs based on the same file name.
+
+
+```shell
+pip install gdown
+pip install finetuner
+pip install torchvision
+
+gdown https://drive.google.com/uc?id=1jvkbTr_giSP3Ru8OwGNCg6B4PvVbcO34
+gdown https://drive.google.com/uc?id=1EzBZUb_mh_Dp_FKD0P4XiYYSd0QBH5zW
+
+unzip left.zip
+unzip right.zip
+```
+
+Afterward, we load all images from unzipped `left` and `right` folders and turn them into sorted order as jina `DocumentArray`.
+80% of the dataset will be used to train the machine learning model.
+While 20% of the dataset will be used to evaluate the quality of embeddings on the search task.
+
+```python
+from docarray import DocumentArray
+
+left_da = DocumentArray.from_files('left/*.jpg')
+right_da = DocumentArray.from_files('right/*.jpg')
+# we use 80% for training machine learning model.
+left_da.sort(key=lambda x: x.uri)
+right_da.sort(key=lambda x: x.uri)
+
+ratio = 0.8
+train_size = int(ratio * len(left_da))
+
+train_da = left_da[:train_size] + right_da[:train_size]
+train_da = train_da.shuffle()
```

Comment:
Shuffling just the train data does not make sense. Can we shuffle the entire set and take a subset of it as training data?

```suggestion

ratio = 0.8
train_size = int(ratio * len(left_da))

# shuffling the left_da and right_da in unison
idx = np.arange(len(left_da))
np.random.shuffle(idx)
left_da = left_da[list(idx)]
right_da = right_da[list(idx)]

train_da = left_da[:train_size] + right_da[:train_size]
```

Reply:
i think it is fine not a big deal

Reply:
@makram93 I'll remove shuffle since finetuner will do the job